devkit/samples/empty/empty.feature.ndjson

```diff
@@ -0,0 +1,18 @@
+{"gherkinDocument":{"comments":[],"feature":{"children":[{"rule":{"children":[{"background":{"description":"","id":"2","keyword":"Background","location":{"column":5,"line":9},"name":"","steps":[{"id":"1","keyword":"Given ","keywordType":"Context","location":{"column":7,"line":10},"text":"I am going to do something"}]}},{"scenario":{"description":"","examples":[],"id":"3","keyword":"Scenario","location":{"column":5,"line":12},"name":"Blank Scenario","steps":[],"tags":[]}}],"description":"","id":"4","keyword":"Rule","location":{"column":3,"line":8},"name":"A Scenario that has a Background","tags":[]}},{"rule":{"children":[{"scenario":{"description":"","examples":[],"id":"5","keyword":"Scenario","location":{"column":5,"line":15},"name":"Blank Scenario 2","steps":[],"tags":[]}}],"description":"","id":"6","keyword":"Rule","location":{"column":3,"line":14},"name":"A Scenario that does not have a Background","tags":[]}}],"description":"  Sometimes we want to quickly jot down a new scenario without specifying any actual steps\n  for what should be executed.\n\n  In this instance we want to stipulate what should / shouldn't run and what the output is","keyword":"Feature","language":"en","location":{"column":1,"line":1},"name":"Empty Scenarios","tags":[]},"uri":"samples/empty/empty.feature"}}
+{"pickle":{"astNodeIds":["3"],"id":"7","language":"en","name":"Blank Scenario","steps":[],"tags":[],"uri":"samples/empty/empty.feature"}}
+{"pickle":{"astNodeIds":["5"],"id":"8","language":"en","name":"Blank Scenario 2","steps":[],"tags":[],"uri":"samples/empty/empty.feature"}}
+{"stepDefinition":{"id":"0","pattern":{"source":"I am going to do something","type":"CUCUMBER_EXPRESSION"},"sourceReference":{"location":{"line":3},"uri":"samples/empty/empty.feature.ts"}}}
+{"testRunStarted":{"timestamp":{"nanos":0,"seconds":0}}}
+{"testCase":{"id":"10","pickleId":"7","testSteps":[{"id":"9"}]}}
+{"testCase":{"id":"12","pickleId":"8","testSteps":[{"id":"11"}]}}
+{"testCaseStarted":{"attempt":0,"id":"13","testCaseId":"10","timestamp":{"nanos":1000000,"seconds":0}}}
+{"testStepStarted":{"testCaseStartedId":"13","testStepId":"9","timestamp":{"nanos":2000000,"seconds":0}}}
+{"testStepFinished":{"testCaseStartedId":"13","testStepId":"9","testStepResult":{"duration":{"nanos":0,"seconds":0},"status":"UNDEFINED"},"timestamp":{"nanos":3000000,"seconds":0}}}
+{"testCaseFinished":{"testCaseStartedId":"13","timestamp":{"nanos":4000000,"seconds":0},"willBeRetried":false}}
+{"testCaseStarted":{"attempt":0,"id":"14","testCaseId":"12","timestamp":{"nanos":5000000,"seconds":0}}}
+{"testStepStarted":{"testCaseStartedId":"14","testStepId":"11","timestamp":{"nanos":6000000,"seconds":0}}}
+{"testStepFinished":{"testCaseStartedId":"14","testStepId":"11","testStepResult":{"duration":{"nanos":0,"seconds":0},"status":"UNDEFINED"},"timestamp":{"nanos":7000000,"seconds":0}}}
+{"testCaseFinished":{"testCaseStartedId":"14","timestamp":{"nanos":8000000,"seconds":0},"willBeRetried":false}}
```

Comment:
This does not look right to me. It looks like `fake-cucumber` is pulling in the `Background` from the previous rule. Otherwise this scenario would not have any steps. I would definitely start with a much simpler test case.

Reply:
I'll resolve this generation later on once we've got the other item merged and released and we know next steps.